docs/users-guide/join.md

```diff
@@ -39,6 +39,12 @@ Your joins can also include multiple conditions to refine your results. Metabase
 
 ![Joining tables on multiple columns](./images/notebook/joining-on-multiple-columns.png)
 
+## Joining with different operators
+
+You can join tables using `=`, `≠`, `>`, `≥`, `<`, or `≤`.
```

Comment:
Initially I thought this meant that you could use symbols for join _types_ (left, outer, etc.) instead of join _conditions_ (brain confusing ⊆ with ≤, maybe?).

Perhaps something like this:
```suggestion
You can join tables on comparison conditions like `=`, `≠`, `>`, `≥`, `<`, or `≤`.
```